[Resolve] Referrals make parsing fail

Having a referral in an index causes the whole index parsing to fail.

`Caused by: java.lang.reflect.InvocationTargetException
	at org.osgi.util.promise.PromiseImpl.getValue(PromiseImpl.java:159)
	at aQute.bnd.repository.osgi.OSGiIndex.getBridge(OSGiIndex.java:127)
	at aQute.bnd.repository.osgi.OSGiIndex.findProviders(OSGiIndex.java:196)
	at aQute.bnd.repository.osgi.OSGiRepository.findProviders(OSGiRepository.java:329)
	at biz.aQute.resolve.BndrunResolveContext.findRepositoryAugments(BndrunResolveContext.java:419)
	at biz.aQute.resolve.BndrunResolveContext.loadRepositories(BndrunResolveContext.java:394)
	at biz.aQute.resolve.BndrunResolveContext.init(BndrunResolveContext.java:136)
	... 29 more
Caused by: java.lang.NullPointerException
	at java.util.AbstractCollection.addAll(AbstractCollection.java:343)
	at aQute.bnd.osgi.repository.ResourcesRepository.addAll(ResourcesRepository.java:63)
	at aQute.bnd.repository.osgi.OSGiIndex$1.apply(OSGiIndex.java:68)
	at aQute.bnd.repository.osgi.OSGiIndex$1.apply(OSGiIndex.java:1)
	at org.osgi.util.promise.PromiseImpl$Map.call(PromiseImpl.java:409)
	at org.osgi.util.promise.PromiseImpl$Then.run(PromiseImpl.java:240)
	at org.osgi.util.promise.PromiseImpl.notifyCallbacks(PromiseImpl.java:137)
	at org.osgi.util.promise.PromiseImpl.resolve(PromiseImpl.java:119)
	at org.osgi.util.promise.Promises$All.run(Promises.java:174)
	at org.osgi.util.promise.PromiseImpl.notifyCallbacks(PromiseImpl.java:137)
	at org.osgi.util.promise.PromiseImpl.resolve(PromiseImpl.java:119)
	at org.osgi.util.promise.PromiseImpl$Chain.run(PromiseImpl.java:297)
	at org.osgi.util.promise.PromiseImpl.notifyCallbacks(PromiseImpl.java:137)
	at org.osgi.util.promise.PromiseImpl.onResolve(PromiseImpl.java:175)
	at org.osgi.util.promise.PromiseImpl$Then.run(PromiseImpl.java:251)
	at org.osgi.util.promise.PromiseImpl.notifyCallbacks(PromiseImpl.java:137)
	at org.osgi.util.promise.PromiseImpl.resolve(PromiseImpl.java:119)
	at org.osgi.util.promise.Deferred.resolve(Deferred.java:80)
	at aQute.lib.promise.PromiseExecutor$1.run(PromiseExecutor.java:23)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
`

--- a/biz.aQute.bndlib/src/aQute/bnd/osgi/repository/XMLResourceParser.java
+++ b/biz.aQute.bndlib/src/aQute/bnd/osgi/repository/XMLResourceParser.java
@@ -210,6 +210,7 @@ public class XMLResourceParser extends Processor {
 
 			}
 		}
+		next();
 		tagEnd(TAG_REFERRAL);
 	}
 

